package.json

```diff
@@ -81,73 +81,74 @@
     "express-session": "^1.15.6",
     "express-validator": "^4.3.0",
     "extract-text-webpack-plugin": "^3.0.2",
-    "forms": "^1.2.0",
-    "helmet": "^3.1.0",
+    "forms": "^1.3.0",
+    "helmet": "^3.9.0",
     "history": "^4.7.2",
-    "html-pdf": "^2.1.0",
+    "html-pdf": "^2.2.0",
     "ignore-styles": "^5.0.1",
-    "js-cookie": "^2.1.3",
-    "lodash": "^4.15.0",
-    "moment": "^2.15.0",
-    "multer": "^1.2.0",
+    "js-cookie": "^2.2.0",
+    "lodash": "^4.17.4",
+    "moment": "^2.19.2",
+    "multer": "^1.3.0",
     "newrelic": "^2.4.0",
-    "node-sass": "^4.5.2",
-    "node-uuid": "^1.4.7",
+    "node-sass": "^4.7.2",
+    "node-uuid": "^1.4.8",
     "nodemon": "^1.10.2",
     "normalize.css": "^7.0.0",
-    "npm-run-all": "^4.0.2",
+    "npm-run-all": "^4.1.2",
     "object-assign": "^4.1.0",
     "passport": "^0.4.0",
     "passport-unilogin": "^2.0.0",
     "path": "^0.12.7",
-    "pmx": "^1.0.1",
+    "pmx": "^1.5.5",
     "postcss-loader": "^2.0.8",
     "primus": "^7.1.0",
-    "prop-types": "^15.5.10",
-    "proxy-agent": "^2.0.0",
-    "pug": "^2.0.0-beta6",
-    "react": "^15.5.0",
-    "react-addons-test-utils": "^15.6.2",
-    "react-dom": "^15.5.0",
+    "prop-types": "^15.6.0",
+    "proxy-agent": "^2.1.0",
+    "pug": "^2.0.0-rc.4",
+    "react": "^16.1.1",
+    "react-dom": "^16.1.1",
     "react-redux": "^5.0.6",
-    "react-visibility-sensor": "^3.1.2",
-    "react-youtube": "^7.0.1",
+    "react-visibility-sensor": "^3.11.0",
+    "react-youtube": "^7.5.0",
     "redux": "^3.7.2",
     "redux-thunk": "2.2.0",
-    "request": "^2.72.0",
-    "sanitize-html": "^1.13.0",
+    "request": "^2.83.0",
+    "sanitize-html": "^1.16.1",
     "sass-loader": "^6.0.6",
     "sass-mediaqueries": "^1.6.1",
-    "socketcluster": "^5.0.14",
-    "superagent": "^3.1.0",
+    "socketcluster": "^8.0.2",
+    "superagent": "^3.8.1",
     "susy": "2.2.12",
     "svg-inline-loader": "^0.8.0",
     "svg-sprite-loader": "^0.0.31",
-    "teaspoon": "^6.6.0",
-    "twemoji": "^2.1.0",
+    "twemoji": "^2.5.0",
     "vhost": "3.0.2",
     "walk": "2.3.9",
     "webpack": "^3.8.1",
-    "ws": "^2.0.0",
+    "ws": "^3.3.2",
     "xmldom": "0.1.27",
     "xss-filters": "1.2.7",
     "youtube-link-to-id": "^1.0.2"
   },
   "devDependencies": {
     "babel-eslint": "^8.0.2",
+    "babel-preset-airbnb": "^2.4.0",
     "chai": "^4.1.2",
     "chromedriver": "2.30.1",
     "cucumber": "^1.0.0",
     "cucumberjs-junitxml": "1.0.0",
+    "enzyme": "^3.2.0",
+    "enzyme-adapter-react-16": "^1.1.0",
     "eslint": "^4.11.0",
     "eslint-loader": "^1.0.0",
-    "eslint-plugin-react": "^7.4.0",
+    "eslint-plugin-react": "^7.5.1",
     "estraverse": "^4.2.0",
     "estraverse-fb": "^1.3.1",
     "expect": "^21.2.1",
```

Comment:
Kan vi fjerne den?

Reply:
Har fjernet expect helt til fordel for chai's expect.